src/universal/modules/outcomeCard/components/OutcomeCardAssignMenuRoot.js

```diff
@@ -18,41 +17,41 @@ const query = graphql`
   }
 `;
 
+const loading = (props) => <LoadableLoading {...props} height={DEFAULT_MENU_HEIGHT} width={DEFAULT_MENU_WIDTH} />;
+
+const LoadableOutcomeCardAssignMenu = Loadable({
+  loader: () => System.import(
+    /* webpackChunkName: 'OutcomeCardAssignMenu' */
+    'universal/modules/outcomeCard/components/OutcomeCardAssignMenu/OutcomeCardAssignMenu'
+  ),
+  loading,
+  delay: HUMAN_ADDICTION_THRESH,
+  timeout: MAX_WAIT_TIME
+});
 
 const OutcomeCardAssignMenuRoot = (props) => {
-  const {area, atmosphere, closePortal, project, teamId} = props;
+  const {area, assignRef, atmosphere, closePortal, project, teamId} = props;
   return (
     <QueryRenderer
       cacheConfig={cacheConfig}
       environment={atmosphere}
       query={query}
       variables={{teamId}}
-      render={({error, props: renderProps}) => {
-        return (
-          <TransitionGroup appear component={null}>
-            {error && <ErrorComponent height={'14rem'} error={error} />}
-            {renderProps && <AnimatedFade key="1">
-              <OutcomeCardAssignMenu
-                area={area}
-                closePortal={closePortal}
-                project={project}
-                viewer={renderProps.viewer}
-              />
-            </AnimatedFade>}
-            {!renderProps && !error &&
-            <AnimatedFade key="2" unmountOnExit exit={false}>
-              <LoadingComponent height={'5rem'} />
-            </AnimatedFade>
-            }
-          </TransitionGroup>
-        );
-      }}
+      render={(readyState) => (
+        <RelayLoadableTransitionGroup
+          LoadableComponent={LoadableOutcomeCardAssignMenu}
+          loading={loading}
```

Comment:
reusing the react-loadable loading component for relay loading. end result is a really clean open/close modal with no jank.